perpVects/MakefileDev

```diff
@@ -1,12 +1,12 @@
 #CC := clang++
-CC := g++
+CC := g++-4.9
```

Comment:
Yes, it is for CUDA compatibility.

If there're features you'd like to use not supported with 4.9, including c++1y (4|7), feel free. Just provide alternatives via a macro, '__CUDA__' is a good choice.